.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "airbnb",
   "rules": {
+    "import/prefer-default-export": "off",
```

Comment:
Hmm, how does it work if you don't have eslint in dependencies?

Reply:
It works fine) At least for me. It just forces a user to use a default export instead of a named one. According to the task, we have to create named classes. And it was a bit annoying to see red linting near the export keyword all the time.